editor/src/components/inspector/sections/layout-section/flex-element-subsection/flex-element-subsection.spec.browser2.tsx

```diff
@@ -0,0 +1,277 @@
+import { CanvasControlsContainerID } from '../../../../../components/canvas/controls/new-canvas-controls'
+import { mouseClickAtPoint } from '../../../../../components/canvas/event-helpers.test-utils'
+import { cmdModifier } from '../../../../../utils/modifiers'
+import {
+  EditorRenderResult,
+  getPrintedUiJsCode,
+  renderTestEditorWithCode,
+} from '../../../../../components/canvas/ui-jsx.test-utils'
+import {
+  AbsoluteLengthUnits,
+  AllFlexDirections,
+  FlexDirection,
+  LengthPercentUnit,
+  LengthPercentUnits,
+  LengthUnit,
+  LengthUnits,
+} from '../../../../../components/inspector/common/css-utils'
+import { act, fireEvent } from '@testing-library/react'
+import { applyPrettier } from 'utopia-vscode-common'
+
+function downClickUp(targetElement: HTMLElement, mouseEventInit: MouseEventInit) {
+  fireEvent(targetElement, new MouseEvent('mousedown', mouseEventInit))
+  fireEvent(targetElement, new MouseEvent('click', mouseEventInit))
+  fireEvent(targetElement, new MouseEvent('mouseup', mouseEventInit))
+}
```

Comment:
`mouseClickAtPoint` from event-helpers.test-utils.ts should do the job here (it will also ensure those events are fired in the right order), unless you specifically didn't want to provide the mouse point here? If that's a suitable use case then maybe we should address that in event-helpers.test-utils.ts

Reply:
I did have that in there originally, this is a victim of me having some grief with the input box itself. Will look into restoring that now.

Reply:
Fixed in 5d8f5a6a5.